README.md

```diff
@@ -324,6 +324,53 @@ query_config = QueryConfig() # Currently no options
 print(naval_chat_bot.query("What unique capacity does Naval argue humans possess when it comes to understanding explanations or concepts?", query_config))
 ```
 
+Here's the example of using custom prompt template with `.query`
+
+```python
+from embedchain.config import QueryConfig
+from embedchain.embedchain import App
+from string import Template
+import wikipedia
+
+einstein_chat_bot = App()
+
+# Embed Wikipedia page
+page = wikipedia.page("Albert Einstein")
```

Comment:
first of all, you're using Einstein here, nothing wrong with that, just @taranjeet  said before that he wants all examples to be about Naval.

Secondly, I'm not sure if we want the complexity. of adding another import `wikipedia` just for this example

Reply:
1) Should I try to add the same example for Naval instead of Einstein?
2) I added `wikipedia` as it made the model answer more accurately. I tried adding the wikipedia link as a `web_page` but it didn't really gave accurate results, as the web_page was not parsed correctly. Should I try a simpler example?

PS: I also believe support for Wikipedia should be inherent to `embedchain`. It would make it more useful for someone to research a subject through wikipedia. Maybe we could add `wikipedia` as a `data_type`. Just a suggestion.

Reply:
> Maybe we could add wikipedia as a data_type. Just a suggestion.

Yes I agree. 

> Should I try to add the same example for Naval instead of Einstein?

up to @taranjeet

Reply:
> Maybe we could add wikipedia as a data_type. Just a suggestion.

We should definitely do that. @ianupamsingh : do you want to open an issue for this?

Reply:
> > Maybe we could add wikipedia as a data_type. Just a suggestion.
> 
> We should definitely do that. @ianupamsingh : do you want to open an issue for this?

Issue created: https://github.com/embedchain/embedchain/issues/212

Reply:
thanks